<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAlarm</class>
 <widget class="QDialog" name="AddAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url();
background-color: rgb();</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>601</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>155</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(32, 90, 160);
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/33318.ico</normaloff>:/image/33318.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(32, 90, 160);
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/ok.ico</normaloff>:/image/ok.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="alarmTime">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(32, 90, 160);
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/add.ico</normaloff>:/image/add.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMinus">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(32, 90, 160);
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/minus.ico</normaloff>:/image/minus.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>120</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(32, 90, 160);
border: 2px groove gray;
border-radius: 10px;
padding: 2px 4px;</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
     <property name="icon">
      <iconset resource="ikembx00.qrc">
       <normaloff>:/image/ui_edit_delete.ico</normaloff>:/image/ui_edit_delete.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="labTimeIcon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>63</y>
       <width>58</width>
       <height>58</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ikembx00.qrc">:/image/alarmclock.ico</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>155</y>
      <width>411</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 48, 180);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>设置闹铃</string>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ikembx00.qrc"/>
 </resources>
 <connections/>
</ui>
